tools/cli/internal/openapi/filter/path.go

```diff
@@ -257,6 +259,13 @@ func filterVersionedContent(content map[string]*openapi3.MediaType, version *api
 	return nil, nil
 }
 
+func updateSingleMediaTypeExtension(m *openapi3.MediaType, version *apiversion.APIVersion) {
```

Comment:
Could you add a small description of what the functions does?

Reply:
thats trash, deleted now!

Reply:
oh no, nvm thought it was a comparison one, will do!